lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -91,6 +102,19 @@ def test_invalid_course_id(self):
         self.form_data["course_id"] = "invalid course id"
         self.assert_error("course_id", "'invalid course id' is not a valid course id")
 
+    def test_empty_topic_id(self):
+        self.form_data.setlist("topic_id", ["", "not empty"])
+        form = self.get_form(expected_valid=True)
+        self.assertEqual(
+            form.cleaned_data,
+            {
+                "course_id": CourseLocator.from_string("Foo/Bar/Baz"),
+                "page": 2,
+                "page_size": 13,
+                "topic_id": ["", "not empty"],
```

Comment:
We probably shouldn't allow an empty topic_id. Can you check whether the comments service handles that correctly?